dashboard/db/migrate/20170522221144_add_course_id_to_sections.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseIdToSections < ActiveRecord::Migration[5.0]
+  def change
+    add_column :sections, :course_id, :integer
```

Comment:
Looking to confirm that we don't need an index on `course_id`, just as we don't have an index on `script_id`.